src/components/LandDetailPage/LandDetailPage.tsx

```diff
@@ -185,7 +185,7 @@ export default class LandDetailPage extends React.PureComponent<Props, State> {
                       </Dropdown>
                     </Row>
                   </Column>
-                ) : land.roles.includes(RoleType.TENANT) ? (
+                ) : land.roles.includes(RoleType.TENANT) && rental && rental.endsAt.getTime() > Date.now() ? (
```

Comment:
np: this could be moved to something like this: `const isRentalEnd = (rental) => rental && rental.endsAt.getTime() > Date.now()`

Reply:
That's a great idea, changed it!